Cargo.toml

```diff
@@ -26,10 +26,11 @@ workspace = true
 
 # See https://github.com/johnthagen/min-sized-rust for additional optimizations
 [profile.release]
-debug = 0
-lto = "fat"
+debug = false
+lto = true
 opt-level = 3     # Optimize for speed, not size.
 codegen-units = 1 # Reduce parallel code generation.
+panic = "abort"
```

Comment:
Besides the impact on compilation speed, do these changes affect the performace of the compiled code like speed or binary size?

Reply:
The debug and lto aren't real changes (just using different names for the same thing), but the panic abort is supposed to increase performance.